src/messages/layouts/MessageLayoutContainer.hpp

```diff
@@ -86,7 +91,8 @@ struct MessageLayoutContainer {
     };
 
     // helpers
-    void _addElement(MessageLayoutElement *element, bool forceAdd = false);
+    void _addElement(MessageLayoutElement *element, bool forceAdd = false,
```

Comment:
Could you incorporate your comment from here https://github.com/Chatterino/chatterino2/pull/3958#issuecomment-1249849323 to this function?